snykTask/src/__tests__/install/index.test.ts

```diff
@@ -102,11 +123,15 @@ describe('getSnykDownloadInfo', () => {
     expect(dlInfo).toEqual({
       snyk: {
         filename: 'snyk-macos',
-        downloadUrl: 'https://static.snyk.io/cli/v1.1287.0/snyk-macos',
+        downloadUrl:
+          'https://downloads.snyk.io/cli/v1.1287.0/snyk-macos?utm_source=AZURE_DEVOPS',
```

Comment:
Small Issue: For consistency, Let's use the same name for utm_source as for integration name `AZURE_PIPELINES` please.